sdk/keyvault/azure-security-keyvault-administration/vcpkg.json

```diff
@@ -1,6 +1,3 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# SPDX-License-Identifier: MIT
```

Comment:
IIRC our vcpkg publishing pipeline strips these off, but in general we keep them.

Reply:
From what I can tell (and FWIW), that's for all the `vcpkg/vcpkg.json` files (the vcpkg.json file nested in the vcpkg/ folder for each service). The other "root" vckpg.json files don't have this comment.

Reply:
No we do not keep these comments, check the other folders , this is the root vcpkg.json for the package not the one in the vcpkg folder.

Reply:
even the attestation one is missing thhe header, the build actually fails if the comment is there vcpkg complkains that it is invalid json